src/operations/get-read-list-entity.handler.ts

```diff
@@ -1,7 +1,12 @@
 import { Request, RequestHandler, Response } from 'express';
 import { DataAdapterStorage } from '../utils/create-data-adapter-storage';
 import { ReadListOperationConfig } from '../typing-utils/operations';
-import { matchEntitiesByBodyFilters, matchEntitiesByParams, matchEntitiesByQueryFilters } from '../utils/entity-utils';
+import {
+  matchEntitiesByBodyFilters,
+  matchEntitiesByFilters,
```

Comment:
let's get rid of general filters. I think it's too soon to think about backward compatibility.